backend/requirements.txt

```diff
@@ -1,6 +1,6 @@
 pandas
 networkx
-numpy
+numpy==1.19.3
```

Comment:
why?

Reply:
With the newest version of numpy project couldn't build for me. I read that it's because for now Python3.9 isn't compatible with numpy 1.19.4

Reply:
OK, let's keep it that way since we prefer to support newer versions of Python over NumPy.